hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/snapshot/OMSnapshotCreateRequest.java

```diff
@@ -225,6 +224,7 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
     if (exception == null) {
       LOG.info("Created snapshot '{}' under path '{}'",
           snapshotName, snapshotPath);
+      omMetrics.incNumSnapshotActive();
```

Comment:
Would it not be better to write a unit testcase for this? As far as I understand this should be still a smaller test & runtime of a unit test would be faster.

Reply:
Agreed and move to unit tests instead of integ tests.